src/Sarif.Multitool.Library/Rules/RuleResources.resx

```diff
@@ -452,12 +452,18 @@ Semantics: Assuming the reader of the log file (an end user or another tool) has
     <value>{0}: The tool '{1}' does not provide 'informationUri'. This property helps the developer responsible for addessing a result by providing a way to learn more about the tool.</value>
   </data>
   <data name="SARIF2012_ProvideRuleProperties_Note_ProvideFriendlyName_Text" xml:space="preserve">
-    <value>{0}: The rule '{1}' does not provide a "friendly name" in its 'name' property. The friendly name should be a single Pascal identifier, for example, 'ProvideRuleFriendlyName', that helps users see at a glance the purpose of the analysis rule.</value>
+    <value>{0}: The rule '{1}' does not provide a "friendly name" in its 'name' property. The friendly name should be a single Pascal-case identifier, for example, 'ProvideRuleFriendlyName', that helps users see at a glance the purpose of the analysis rule.</value>
   </data>
   <data name="SARIF2012_ProvideRuleProperties_Note_FriendlyNameNotAPascalIdentifier_Text" xml:space="preserve">
-    <value>{0}: '{1}' is not a Pascal identifier. For uniformity of experience across all tools that produce SARIF, the friendly name should be a single Pascal identifier, for example, 'ProvideRuleFriendlyName'.</value>
+    <value>{0}: '{1}' is not a Pascal-case identifier. For uniformity of experience across all tools that produce SARIF, the friendly name should be a single Pascal-case identifier, for example, 'ProvideRuleFriendlyName'.</value>
   </data>
   <data name="SARIF1004_ExpressUriBaseIdsCorrectly_Error_RelativeReferenceMustNotBeginWithSlash_Text" xml:space="preserve">
     <value>The relative reference '{0}' begins with a slash, which will prevent it from combining properly with the absolute URI specified by a 'uriBaseId'.</value>
   </data>
+  <data name="SARIF2012_ProvideRuleProperties_Note_ProvideMetadataForAllViolatedRules_Text" xml:space="preserve">
+    <value>'{0}' does not provide a 'rules' property. 'rules' contain information that helps users understand why each rule fires and what the user can do to fix it.</value>
```

Comment:
>rules [](start = 37, length = 5)

Feel free to reach out to Larry if you'd like another set of eyes on this text proposal.

Reply:
LGTM

---
In reply to: [533635662](https://github.com/microsoft/sarif-sdk/pull/2161#discussion_r533635662) [](ancestors = 533635662)